Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -157,7 +157,7 @@ class DataPipelineImplementation: DataPipelineInstance {
         let isFirstTimeIdentifying = currentlyIdentifiedProfile == nil
 
         if isChangingIdentifiedProfile, let _ = registeredDeviceToken {
-            logger.debug("deleting registered device token from existing profile: \(currentlyIdentifiedProfile ?? "nil")")
+            dataPipelinesLogger.logDeletingTokenDueToNewProfileIdentification()
```

Comment:
think its still helpful to print whats the currentlyIdentifiedProfile at the moment we are about to delete token? so its easier to know which user id shouldnt have the token anymore